docs/ops.md

```diff
@@ -31,63 +31,47 @@ You will be raising a Pull Request from your fork to the upstream repo.
 
 We recommend only adding one chain at a time, and starting with a fresh branch of this repo for every chain.
 
-### 3. Stage your state file
-
-Move the `state.json` generated by `op-deployer` into the `.staging` directory. This will allow the validation tooling to find out information about you r chain and its deployment parameters.
-
-### 4. Fill out metadata template
-
-Copy the below template into a file named `metadata.toml`, and put it in the `.staging` directory. This will provide information about your chain's name, RPC endpoints, governance status, and more. All fields are required:
-
-```toml
-# Human-readable name for your chain
-name = "SlipChain"
-# Machine-readable name for your chain. Config filenames will be named after this. Can only contain alphanumeric characters, dashes, and underscores.
-short_name = "slipchain"
-# Which superchain your chain belongs to. Can by sepolia or mainnet.
-superchain = "sepolia"
-# Public RPC endpoint for your chain.
-public_rpc = "https://foo.bar.net"
-# Sequencer RPC endpoint for your chain.
-sequencer_rpc = "https://foo.bar.net"
-# Block explorer for your chain.
-explorer = "https://foo.bar.net"
-# Superchain level for your chain.
-superchain_level = 0
-# Whether or not your chain is governed by Optimism. Must be `false`.
-governed_by_optimism = false
-# UNIX timestamp after which your chain will be opted into Superchain-wide upgrades.
-superchain_time = 0
-# DA type. Should probably be eth-da.
-data_availability_type = "eth-da"
-# Transaction hash of the transaction OP Deployer generated to deploy your chain.
-deployment_tx_hash = ""
-```
-
-### 5. Generate code
+### 3. Generate a config from your state file
 
-Run the following script from the root of the repository to open the PR.
+From the root of this directory, run the following:
 
-```shell
-just sync-staging
 ```
+just inflate-config <shortname> <path to your state file>
+```
+
+This will put two files in the `.staging` directory - `<shortname>.toml` and `<shortname>.json.zst`
 
-The team will then review and merge your chain when it's ready.
+### 4. Update generated config
 
-### 6. Understand output
+Update `<shortname>.toml` with your chain's information. Most of the data is automatically populated from your deployer
+state file, however you will need to populate the following yourself:
 
-The tool will write the following data:
+- `name`: Human-readable name for your chain.
```

Comment:
Do we still validate that this name matches the one stored at "https://chainid.network/chains_mini.json"? If so, it would be good to mention that here

Reply:
It does not do this validation - I wasn't aware that was a requirement. I can bring it back if needed.

Reply:
Not sure that its a requirement from the standard charter. But its a nice sanity check to ensure the superchain-registry data matches chainid's data. Previously we required the chain to be added to chainid.network first, then we verified the data matched what was being added to the superchain-registry.

I see that you are still grabbing the data from chainid.network as part of the `FetchGlobalChainIDs` function, but it seems the test will fail if the chain already exists at chainid.network? Is that intentional?

Reply:
> Is that intentional?

Yes, but I didn't realize that there was a requirement that the chain already exist in the global chain IDs. I can update the code to fix that.

Reply:
fixed, there's a new CI check for it